src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/config/ControllerAdvice.kt

```diff
@@ -0,0 +1,136 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.config
+
+import org.slf4j.LoggerFactory
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.security.access.AccessDeniedException
+import org.springframework.security.access.AuthorizationServiceException
+import org.springframework.web.bind.annotation.ExceptionHandler
+import org.springframework.web.bind.annotation.RestControllerAdvice
+import org.springframework.web.client.RestClientException
+import org.springframework.web.client.RestClientResponseException
+// import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler
```

Comment:
unused import - dont we have the lint check enabvled yet? should pick out things like this

Reply:
I've updated this now. As it was commented, the linter ignores - it would pick it up if uncommented and unused.